libraries/libmesosphere/source/board/nintendo/switch/kern_secure_monitor.hpp

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2018-2019 Atmosphère-NX
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms and conditions of the GNU General Public License,
+ * version 2, as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+#include <mesosphere.hpp>
+
+namespace ams::kern::smc {
+
+    /* Types. */
+    enum MemoryMode {
+        MemoryMode_4GB = 0,
+        MemoryMode_6GB = 1,
+        MemoryMode_8GB = 2,
+    };
+
+    enum class ConfigItem : u32 {
```

Comment:
Maybe double check the names here and decide a final scheme (e.g.: NewHardwareType is RegulatorType).

Reply:
Good call :)